administrator/components/com_joomgallery/models/category.php

```diff
@@ -948,6 +1010,14 @@ protected function batchMove($value, $pks)
         return false;
       }
 
+      // Trigger Event onJoomBeforeSave (Returnvalue: true or false)
+      // $table contains still the old values
+      $plugins = $this->_mainframe->triggerEvent('onJoomBeforeSave', array(_JOOM_OPTION.'.category', $table, false, array('parent_id'=>$categoryId)));
+      if(in_array(false, $plugins, true))
+      {
+        return false;
+      }
+
```

Comment:
Once again a question. Isn't it necessary to additionally trigger 'onContentBeforeSave' for the JoomFinderPlugin somewhere here in the function?